src/components/ProgramRecord/ProgramRecord.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useState, useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+
+import {
+  Info, ArrowBack,
+} from '@edx/paragon/icons';
+import {
+  Alert, Container, Button, Hyperlink,
+} from '@edx/paragon';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+import _ from 'lodash';
+
+import ProgramRecordActions from './ProgramRecordActions';
+import ProgramRecordHeader from './ProgramRecordHeader';
+import ProgramRecordTable from './ProgramRecordTable';
+import RecordsHelp from './RecordsHelp';
+import ProgramRecordAlert from '../ProgramRecordAlert';
+import SendLearnerRecordModal from '../ProgramRecordSendModal';
+
+import getProgramDetails from './data/service';
+
+const ProgramRecord = ({ isPublic }) => {
+  const [recordDetails, setRecordDetails] = useState({});
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [hasNoData, setHasNoData] = useState(false);
+  const [showSendRecordButton, setShowSendRecordButton] = useState(false);
+
+  const [sendRecord, setSendRecord] = useState({
+    sendRecordModalOpen: false,
+    sendRecordSuccessOrgs: [],
+    sendRecordFailureOrgs: [],
+  });
+
+  const { programId } = useParams();
+
+  useEffect(() => {
+    getProgramDetails(programId, isPublic).then((data) => {
+      if (_.isEmpty(data)) {
+        setHasNoData(true);
+      } else {
+        setRecordDetails(data);
+        setShowSendRecordButton(data.record.pathways.length > 0);
+      }
+      setIsLoaded(true);
+    }).catch((error) => {
+      const errorMessage = (`Error: Could not fetch program record data for user: ${error.message}`);
+      logError(errorMessage);
+    });
+  }, [programId, isPublic]);
+
+  const toggleSendRecordModal = () => {
+    setSendRecord(prev => ({
+      ...prev,
+      sendRecordModalOpen: !sendRecord.sendRecordModalOpen,
+    }));
+  };
+
+  const updateSuccessAndFailureOrgs = (id) => {
+    setSendRecord(prev => ({
+      ...prev,
+      sendRecordSuccessOrgs: prev.sendRecordSuccessOrgs.filter(org => org.id !== id),
+      sendRecordFailureOrgs: prev.sendRecordFailureOrgs.filter(org => org.id !== id),
+    }));
+  };
+
+  const renderBackButton = () => (
+    <Button
+      variant="tertiary"
+      iconBefore={ArrowBack}
+      className="back-to-records"
+    >
+      <Hyperlink
+        destination="/"
```

Comment:
I feel like I might be a lil dumb, what does this do? Sends you back to `records.edx.org`? I figured it had to be more, well, explicit in the destination?

Reply:
No I think you're right. It works locally but it looks like it should probably reference the `BASE_URL` environment variable